build/configs/artik053/audio/defconfig

```diff
@@ -423,9 +423,10 @@ CONFIG_AUDIO_ALC5658=y
 CONFIG_ALC5658_INITVOLUME=250
 CONFIG_ALC5658_INFLIGHT=2
 CONFIG_ALC5658_MSG_PRIO=1
-CONFIG_ALC5658_BUFFER_SIZE=8192
-CONFIG_ALC5658_NUM_BUFFERS=4
+CONFIG_ALC5658_BUFFER_SIZE=6400
+CONFIG_ALC5658_NUM_BUFFERS=2
```

Comment:
Framesize should be calculated how many interrupt processed in 1sec.
basically our max time out value is 100ms, I just calculated using this value.
since we decided 16k / 2ch / 2bytes as a default setting, defconfig value should be aligned with it.
This values can be changed anytime by requirement, currently we'd better keep it.
Let me hear if you have diffrent opinion.